pom.xml

```diff
@@ -42,9 +42,9 @@
         <jdk.min.version>11</jdk.min.version>
 
         <!-- Dependency Versions -->
-        <version.postgresql.driver>42.5.1</version.postgresql.driver>
-        <version.com.google.protobuf>3.21.7</version.com.google.protobuf>
-        <version.kafka>3.6.1</version.kafka>
+        <version.postgresql.driver>42.7.2</version.postgresql.driver>
+        <version.com.google.protobuf>3.25.5</version.com.google.protobuf>
+        <version.kafka>3.4.0</version.kafka>
```

Comment:
is it a typo? why downgrading?

Reply:
It wasn't a typo. Snyk suggested me that the only version this vulnerability has been fixed is 3.4.0 so I changed it accordingly. However, upon looking at the releases on maven repository, it looks like 3.7.0 should do as well.